sirepo/package_data/static/html/simulations.html

```diff
@@ -34,7 +34,7 @@
           </thead>
           <tbody>
             <tr data-ng-repeat="item in simulations.activeFolder.children | orderBy:['isFolder', simulations.sortField]">
-              <td><a href data-ng-click="simulations.openItem(item)"><span class="glyphicon" data-ng-class="{'sr-user-item': ! simulations.fileManager.isItemExample(item), 'glyphicon-folder-close': item.isFolder, 'glyphicon-file': ! item.isFolder, 'sr-transparent-icon': ! item.isFolder && item.isExample, 'sr-transparent-icon-user': ! item.isFolder && ! item.isExample}"></span> <span data-ng-class="{ 'sr-user-item': ! simulations.fileManager.isItemExample(item) }">{{ item.name | simulationName }}</span></a></td>
+              <td><a data-ng-href="{{nav.simulationURL(item.simulationId)}}" data-ng-click="simulations.openItem(item)"><span class="glyphicon" data-ng-class="{'sr-user-item': ! simulations.fileManager.isItemExample(item), 'glyphicon-folder-close': item.isFolder, 'glyphicon-file': ! item.isFolder, 'sr-transparent-icon': ! item.isFolder && item.isExample, 'sr-transparent-icon-user': ! item.isFolder && ! item.isExample}"></span> <span data-ng-class="{ 'sr-user-item': ! simulations.fileManager.isItemExample(item) }">{{ item.name | simulationName }}</span></a></td>
```

Comment:
Spaces after `{{` and before `}}`. I call this "leave no traces coding". The indentation, spacing, brace placement, etc. should all match the existing code. We don't have a formatter for this type of code (please suggest one if one exists). 

```suggestion
              <td><a data-ng-href="{{ nav.simulationURL(item.simulationId) }}" data-ng-click="simulations.openItem(item)"><span class="glyphicon" data-ng-class="{'sr-user-item': ! simulations.fileManager.isItemExample(item), 'glyphicon-folder-close': item.isFolder, 'glyphicon-file': ! item.isFolder, 'sr-transparent-icon': ! item.isFolder && item.isExample, 'sr-transparent-icon-user': ! item.isFolder && ! item.isExample}"></span> <span data-ng-class="{ 'sr-user-item': ! simulations.fileManager.isItemExample(item) }">{{ item.name | simulationName }}</span></a></td>
```

Reply:
Thanks. I've used [prettier](https://github.com/prettier/prettier) on all other personal and professional JS projects. I personally really enjoy it but I also like opinionated tools. I've never used it with Angular templates. They seem to be [supported](https://prettier.io/blog/2018/11/07/1.15.0.html) but only since Nov 7, 2018. The React JSX support was very good.

Reply:
It is commonly used in conjunction with eslint which I also like using. It looks like there is some Angular [support](https://github.com/Gillespie59/eslint-plugin-angular).

Reply:
We use jshint, but I don't think anybody is attached. If we could use something whose config works with PyCharm and Emacs, it would be great. This could be a project if you are interested. Not the highest priority, obviously, since there's only a few of us.